lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -82,7 +82,7 @@ export class ServerRequestParameters {
      * @param request
      * @param serverAuthenticationRequest
      */
-    populateQueryParams(account: Account, request: AuthenticationParameters, adalIdTokenObject?: object, silentCall?: boolean): void {
+    populateQueryParams(account: Account, request: AuthenticationParameters|null, adalIdTokenObject?: object, silentCall?: boolean): void {
```

Comment:
I don't think we need to type things with ```|null``` since we don't have ```strictNullChecks``` enabled right? Seems like something we should get around to since it keeps coming up, but should enabling this and fixing the types be its own PR?

Reply:
Yeah, you're right, but I was fine with it since we know want to do it and it doesn't cause any harm.